services/server/config/config.go

```diff
@@ -41,6 +41,8 @@ type Config struct {
 	State string `toml:"state"`
 	// PluginDir is the directory for dynamic plugins to be stored
 	PluginDir string `toml:"plugin_dir"`
+	// WindowsPriorityClass is the priority class used to run containerd and child processes at on Windows machines
+	WindowsPriorityClass string `toml:"windows_priority_class"`
```

Comment:
Add a blurb about what the supported values for this would be, e.g. the subset of options here https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-setpriorityclass

Reply:
done